meson.build

```diff
@@ -90,6 +90,7 @@ newlib_initfini = get_option('newlib-initfini')
 thread_local_storage = get_option('thread-local-storage')
 tls_model = get_option('tls-model')
 posix_io = get_option('posix-io')
+sysroot_install = get_option('sysroot-install')
```

Comment:
I don't see any uses of this new option; it sounds like it's not necessary?

Reply:
This option select install_prefix or "%R" prefix in specfile. This is necesary for non absolute PATH work,